backend/app/services/dingtalk_wikispace_service.py

```
@@ -443,19 +427,22 @@ async def _fetch_all_wikispace_nodes(
+                        nodes_for_kb: list[dict[str, Any]] = []
                         await DingTalkWikiSpaceService._list_nodes_in_wikispace(
                             session=session,
                             workspace_id=kb_id,
-                            all_nodes=all_nodes,
+                            all_nodes=nodes_for_kb,
                         )
+                        all_nodes.extend(nodes_for_kb)
                         # Only add the KB root folder AFTER successfully listing contents.
                         all_nodes.append(kb_as_folder)
-                    except Exception as exc:
+                    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)